web/src/engine/osk/src/views/floatingOskView.ts

```diff
@@ -64,6 +64,7 @@ export default class FloatingOSKView extends OSKView {
     this.resizeBar.on('showbuild', () => this.emit('showbuild'));
 
     this.headerView = this.titleBar;
+    this._Box.insertBefore(this.headerView.element, this._Box.firstChild);
```

Comment:
I presume this has to be insertBefore rather than appendChild because it's happening at a different point in the OSK construction process now?

Reply:
Yes.

For one, it's actually happening _during construction_, rather than when a keyboard is being loaded.

Secondly, this happens after `super.init()`, and that means this insertion is being done after the banner-view and base (empty) keyboard-view have been inserted during base-class construction.